docs/getting-started/tools-for-getting-started-with-the-us-forms-system.md

```diff
@@ -0,0 +1,36 @@
+# Tools for getting started with the US Forms System
+
+Building a form using the US Forms System requires some minimum front-end development tools, as well as general knowledge of those tools. Where possible, we'll provide links to information about getting started with these tools.
+
+### In this guide
+
+- [Command line interface (CLI)](#command-line-interface-cli)
+- [JavaScript](#javascript)
+- [Node.js and npm](#node-js-and-npm)
+- [React](#react)
+- [JSON Schema](#json-schema)
+- [react-jsonschema-form (rjsf)](#react-jsonschema-form-rjsf)
+
+### Command line interface (CLI)
+
+You should be comfortable using command line tools such as Terminal (Mac and Linux) or Command Prompt (Windows). These tools come pre-installed on nearly all computers. There are several tutorials of various types available online to learn about the CLI.
+
+### JavaScript
+
+Most technologies the US Forms System uses, such as React, are centered around JavaScript. We recommend basic familiarity with JavaScript to build forms with the US Forms System. For more information about JavaScript, including tutorials for getting started, see [Pluralsight's JavaScript website](https://www.javascript.com/).
+
+### Node.js and npm
+
+Node.js is a JavaScript runtime environment that allows you to execute complete JavaScript programs. When you install Node.js, it automatically installs a package manager called npm. You need both Node.js and npm to develop using the US Forms System. For more information, including installation instructions, see the [Node.js website](https://nodejs.org).
```

Comment:
I'm not totally positive about this, but I think they need at least Node version 8 in order for this to work properly. I don't know if there's a specific version of npm needed in order for this to work, but I'm using 5.6.0 locally, so I might recommend at least having version 5 or great installed? @dmethvin-gov do you know more about this?

Reply:
Dave says 8 is right, so that's what I documented. 👍